.github/workflows/commit-message-check.yml

```
@@ -0,0 +1,26 @@
+on: [pull_request, push]
+
+jobs:
+  check-commit-message:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get total commits in branch
+        id: total-commits
+        run: |
```

actionlint

"github.event.commits.*.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)